### PR TITLE
Introduces support for explorer page customization

### LINKED ISSUE #2156: Introduce a proper constant for the various explorer pages and expose it to the premium components
## Problem Definition

The urls are hardcoded through various places around the application which make's it harder to maintain, and easier to miss bugs.

### Tasks
- [x] Create a per change mapping of URLs to external block explorers (per chain)
- [x] Expose it to the premium components.

### LINKED ISSUE #2472: Support configurable blockchain explorers
## Abstract

The default blockchain explorer for BTC is blockstream.info

I would like to be able to configure the explorer to another e.g. blockchair.com

## Motivation

Because I almost puked.

### Tasks
- [ ] Show the URL as alt when hovering over the link
- [ ] Add the ability to override the default explorer for each supported chain

### PR DESCRIPTION
**Pull request by kelsos**
Closes #2156 
Closes #2472

### CODE REVIEW

File: frontend/app/src/locales/en.json [hunk 1]
```diff
@@ -153,6 +153,15 @@
     "hint": "Select as the main currency",
     "profit_currency": "Profit Currency"
   },
+  "explorers": {
+    "title": "Blockchain explorers",
+    "address_url": "Addresses will will open at {addressUrl}",
```
Comment (LefterisJP) [inline frontend/app/src/locales/en.json:158]:
```suggestion
    "address_url": "Addresses will open at {addressUrl}",
```

File: frontend/app/src/locales/en.json [hunk 2]
```diff
@@ -153,6 +153,15 @@
...
+    "subtitle": "Set the external explorer urls for addresses and transaction",
+    "tx_url": "Transactions will will open at {txUrl}",
```
Comment (LefterisJP) [inline frontend/app/src/locales/en.json:160]:
```suggestion
    "tx_url": "Transactions will open at {txUrl}",
```

File: frontend/app/tests/unit/store/settings/actions.spec.ts [hunk 3]
```diff
@@ -13,7 +13,8 @@ import {
   THOUSAND_SEPARATOR,
   DECIMAL_SEPARATOR,
   CURRENCY_LOCATION,
-  REFRESH_PERIOD
+  REFRESH_PERIOD,
+  EXPLORERS
```
Comment (LefterisJP) [inline frontend/app/tests/unit/store/settings/actions.spec.ts:17]: so this is also stored in the frontend settings of the user DB, right?

**Review by LefterisJP — COMMENTED** [review]
Seems to work for ethereum stuff that I checked. Some small comments.

File: frontend/app/tests/unit/store/settings/actions.spec.ts (hunk 3 above)
Comment (kelsos) [reply frontend/app/tests/unit/store/settings/actions.spec.ts:17]: Exactly.

**Review by LefterisJP — APPROVED** [review]

**Pull request merged**